docs/Cask-Cookbook.md

```diff
@@ -1131,6 +1131,12 @@ Manual creation can be facilitated with:
 * An uninstaller tool such as [AppCleaner](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/a/appcleaner.rb).
 * Inspection of the usual paths, i.e. `/Library/{'Application Support',LaunchAgents,LaunchDaemons,Frameworks,Logs,Preferences,PrivilegedHelperTools}` and `~/Library/{'Application Support',Caches,Containers,LaunchAgents,Logs,Preferences,'Saved Application State'}`.
 
+If no additional files are discovered, instead of a zap stanza, include the following comment:
+
+```ruby
+# No zap stanza required
+```
```

Comment:
Just a thought, and definitely OOS of this PR, but I wonder if having an explicit 'symbol'/similar in the DSL for 'zap not required' would be good here? eg:

```ruby
zap :not_required
```